PocketKit/Sources/PocketKit/MyList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -447,8 +460,12 @@ extension SavedItemsListViewModel {
 
         if savedItem.shouldOpenInWebView {
             selectedItem = .webView(readable)
+
+            trackContentOpen(destination: .external, item: savedItem)
```

Comment:
Does it make sense to track this `content_open` here, or should we move this type of logic into the view controller(s) that host the offline content?

Reply:
which view controllers are you referring to? This seems okay to me since tracking seems to be at the view model level.